common/formatting/layout_optimizer_test.cc

```diff
@@ -672,59 +672,96 @@ class LayoutFunctionFactoryTest : public ::testing::Test,
             absl::StrSplit(sample_, absl::ByAnyChar(" \n"), absl::SkipEmpty())),
         style_(CreateStyle()),
         factory_(LayoutFunctionFactory(style_)) {
-    for (const auto token : tokens_) {
-      ftokens_.emplace_back(TokenInfo{1, token});
-    }
+    // Create two sets of tokens
+    const ssize_t number_of_tokens_in_set = tokens_.size();
```

Comment:
Windows compile seems to [have a problem](https://github.com/chipsalliance/verible/runs/4898984612?check_suite_focus=true#step:7:684) with the signed `ssize_t` but here we only really need `size_t`.

Reply:
Fixed. Also changed int to size_t in the `for` below.